cfpb/consumerfinance.gov
# Update to DS 3.1.9 - Use filter tags for multiselect tags

## Changes

- Update to DS 3.1.9, which replaces `m-multiselect__choices` with an `m-tag-group` block.


## How to test this PR

1. `yarn install && yarn build` 
2. Visit /blog and see that tags function as before with a slightly looser spacing.


## Screenshots

Before:
<img width="763" alt="Screenshot 2024-09-12 at 10 05 35 AM" src="https://github.com/user-attachments/assets/3b9d94d4-a539-4418-a689-5a5fe325ca9c">

After:
<img width="763" alt="Screenshot 2024-09-12 at 10 05 23 AM" src="https://github.com/user-attachments/assets/e4e624cd-9413-4c52-9bf9-5a442bd741c0">
